### ISSUE TITLE
remove FreeSurfer dependency

### ISSUE THREAD

**Issue by ltirrell** [corticometrics/fs2dicom#8]
Currently FreeSurfer is only used to resample the aseg to native space. can we replace this with nibabel/other means? We are resampling to the dicom file directly, so not sure if we need to create an intermediate nifti first:

```
mri_vol2vol \
  --mov {aseg_file} \
  --targ {t1_dicom_file} \
  --regheader \
  --nearest \
  --o {resampled_aseg}'''
```

**Comment by pwighton** [comment]
Yeah, that's a good idea to swap out `mri_convert` for something like nibabel.  

What do you mean by 'intermediate nifti'? We'll need the resulting labels in something `itkimage2segimage` can read (nifti or nrrd)

**Comment by ltirrell** [comment]
I mean an intermediate T1 nifti (or mgz) file. This isn't necessary for the above FS-based aseg resampling command-- you can point the target at a DICOM file, and it will resample the aseg to that. with nibabel, the target would need[0] to be a nifti/mgz file that it can read in. So the options would be to either make an intermediate T1 nifti file from the DICOM input given by the CLI, or require an additional CLI parameter (the 001.mgz file)

In both cases, the output `resampled_aseg` will still be an `aseg_resampled.nii.gz` file

[0] I didn't do too much investigation on nibabel and DICOM input, so i may be wrong. there is [nicom](http://nipy.org/nibabel/reference/nibabel.nicom.html), but when it is imported within python there's a warning saying that it's experimental and only supports specific DICOMs. Additionally, the [nibabel resampling command](http://nipy.org/nibabel/reference/nibabel.processing.html#nibabel.processing.resample_from_to) could take `(shape, affine)` to resample to, instead of an image object, so maybe this can be derived directly from the DICOM file, without actually using an image

**Comment by pwighton** [comment]
Oh I see, yeah if we could get away from having to use 001.mgz, that would be great!
